images/Makefile

```diff
@@ -19,6 +19,9 @@ PLATFORMS ?= linux_amd64 linux_arm64
 CEPH_BASE_IMAGE ?= ceph/daemon-base:v3.0.5-stable-3.0-luminous-centos-7-$(PLATFORM_ARCH)
 export CEPH_BASE_IMAGE
 
+MINIO_BASE_IMAGE ?= minio/minio
```

Comment:
i think this can actually be moved into the `images/minio` folder. i think the only reason its at this level for ceph is because it's shared by ceph and ceph-toolbox.

Reply:
I'm just going to rip this out. We're not doing anything special to the minio base image, so it's an unnecessary variable.